test/hist.jl

```diff
@@ -2,28 +2,31 @@ using StatsBase
 using Base.Test
 
 # test hist
-@test sum(fit(Histogram,[1,2,3]).weights) == 3
-@test fit(Histogram,Int[]).weights == Int[]
-@test fit(Histogram,[1]).weights == [1]
-@test fit(Histogram,[1,2,3],[0,2,4]) == Histogram([0,2,4],[2,1])
-@test fit(Histogram,[1,2,3],[0,2,4]) != Histogram([0,2,4],[1,1])
-@test fit(Histogram,[1,2,3],0:2:4) == Histogram(0:2:4,[2,1])
-@test all(fit(Histogram,[1:100;]/100,0.0:0.01:1.0).weights .==1)
-@test fit(Histogram,[1,1,1,1,1]).weights[1] == 5
-@test sum(fit(Histogram,(rand(100),rand(100))).weights) == 100
+@test sum(fit(Histogram,[1,2,3], closed=:left).weights) == 3  # FIXME: closed
+@test fit(Histogram,Int[], closed=:left).weights == Int[]  # FIXME: closed
+@test fit(Histogram,[1], closed=:left).weights == [1]  # FIXME: closed
+@test fit(Histogram,[1,2,3],[0,2,4], closed=:left) == Histogram([0,2,4],[1,2], :left)  # FIXME: closed
+@test fit(Histogram,[1,2,3],[0,2,4], closed=:left) != Histogram([0,2,4],[1,1], :left)  # FIXME: closed
+@test fit(Histogram,[1,2,3],0:2:4, closed=:left) == Histogram(0:2:4,[1,2], :left)  # FIXME: closed
+@test all(fit(Histogram,[0:99;]/100,0.0:0.01:1.0, closed=:left).weights .==1)  # FIXME: closed
+@test fit(Histogram,[1,1,1,1,1], closed=:left).weights[1] == 5  # FIXME: closed
+@test sum(fit(Histogram,(rand(100),rand(100)), closed=:left).weights) == 100  # FIXME: closed
 @test fit(Histogram,1:100,nbins=5,closed=:right).weights == [20,20,20,20,20]
 @test fit(Histogram,1:100,nbins=5,closed=:left).weights == [19,20,20,20,20,1]
 @test fit(Histogram,0:99,nbins=5,closed=:right).weights == [1,20,20,20,20,19]
 @test fit(Histogram,0:99,nbins=5,closed=:left).weights == [20,20,20,20,20]
 
-@test fit(Histogram,(1:100,1:100),nbins=5).weights == diagm([20,20,20,20,20])
-@test fit(Histogram,(1:100,1:100),nbins=(5,5)).weights == diagm([20,20,20,20,20])
+# FIXME: closed (all lines in this block):
```

Comment:
I don't remember who came up with it - these are meant to be a reminder to remove `closed = :left` and similar from the code when the deprecation period (during which closed *has* to be specified, so that people will notice the changed default) is over.